### ISSUE TITLE
Glk banners are frequently closed and reopened

### ISSUE THREAD

**Issue by curiousdannii** [tads-intfic/tads-runner#15]
Glk banners are frequently closed and reopened when their content changes. For example, in menus, whenever you move the cursor, both windows are closed then reopened.

Is this behaviour directed from the TADS code? Ie, does the TADS menu delete then reopen all the banners? Or is it a quirk of this Glk implementation? I did notice that the `os_banners_redraw` function does close and reopen them all, so I think it's the latter.

**Comment by realnc** [comment]
I don't believe that the TADS side does that. If that were the case, it would invalidate scroll positions for scrollable banners.

**Comment by angstsmurf** [comment]
I got the impression that it did this in order to update styles on-the-fly, changing the appearance of Glk styles with hints and make the new appearance reflect in the window. I never found an example where this was actually needed, though.

Perhaps somebody intended to use this to make the Glk port support more of the colours and styles of HTML TADS but never got round to it.
